README.md

```diff
@@ -109,8 +109,42 @@ export ETH_SIGNING_KEY=<Your ETH SIGNING KEY>
 pigeon start
 ```
 
-- Remember to run pigeon as a systemd Service! If you have a good systemd service implementation for Ubuntu, please make a PR on this README and we will add it.
-
+#### Using systemd service
+
+```bash
+# create service file
+sudo tee <<EOF >/dev/null /etc/systemd/system/pigeond.service
+[Unit]
+Description=Pigeon daemon
+After=network-online.target
+
+[Service]
+Environment="PALOMA_KEYRING_PASS=<your Paloma key password>"
+Environment="ETH_RPC_URL=<Your Ethereum mainnet RPC URL>"
+Environment="ETH_PASSWORD=<Your ETH Key Password>"
+Environment="ETH_SIGNING_KEY=<Your ETH SIGNING KEY>"
+User=$USER
+ExecStart=/usr/local/bin/pigeon start
+Restart=on-failure
+RestartSec=3
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+# verify the contents
+cat /etc/systemd/system/pigeond.service
+
+# enable
+systemctl enable pigeond
+# reload daemon
+systemctl daemon-reload
```

Comment:
This is required after modifying something in /etc/systemd, I don't think it's required after creating the file. I would delete this whole block and just include

```
service pigeond start
```

(maybe a line after about checking status or logs)

Reply:
Done